The centralized nightlies job failed on Tuesday (2024-11-19)

The centralized nightlies job failed on Tuesday (2024-11-19) in run [11925538829](https://github.com/TileDB-Inc/centralized-tiledb-nightlies/actions/runs/11925538829)

libtiledbvcf [failed](https://github.com/TileDB-Inc/centralized-tiledb-nightlies/actions/runs/11925538829/job/33238332566#step:6:149) to link against libtiledb. This makes sense. Yesterday in #27 we updated everything except libtiledbvcf to use Ubuntu 24 (GCC 13.2.0) instead of Ubuntu 22 (GCC 11.4.0).

```sh
[100%] Linking CXX executable tiledbvcf
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `std::ios_base_library_init()@GLIBCXX_3.4.32'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `__isoc23_strtoull@GLIBC_2.38'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `__isoc23_sscanf@GLIBC_2.38'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `fmodf@GLIBC_2.38'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `__isoc23_strtoll@GLIBC_2.38'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `__isoc23_strtoul@GLIBC_2.38'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_replace_cold(char*, unsigned long, char const*, unsigned long, unsigned long)@GLIBCXX_3.4.31'
/usr/bin/ld: /home/runner/work/centralized-tiledb-nightlies/centralized-tiledb-nightlies/install-libtiledb/lib/libtiledb.so.2.27: undefined reference to `__isoc23_strtol@GLIBC_2.38'
collect2: error: ld returned 1 exit status
```

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -183,7 +183,7 @@ jobs:
       - name: Test
         run: R -e 'tinytest::test_package("tiledb")'
   libtiledbvcf:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     needs: libtiledb
     env:
       LD_LIBRARY_PATH: ${{ github.workspace }}/install-libtiledb/lib

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -202,6 +202,8 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install --yes cmake
+          # Install htslib dependencies
+          sudo apt-get install -y automake autoconf libbz2-dev liblzma-dev
       - name: Configure libtiledbvcf
         run: |
           cmake -S libtiledbvcf -B build-libtiledbvcf \

